.github/workflows/notify-module-version.yml

```diff
@@ -0,0 +1,29 @@
+name: Notify MODULE_VERSION changed
+
+on:
+  push:
+    branches:
+    - master
+    - release-*
+    paths:
+    - kernel-modules/MODULE_VERSION
+
+jobs:
+  notify:
+    runs-on: ubuntu-22.04
```

Comment:
Are there limitations that mean we can't use `ubuntu-latest`?

Reply:
Not really, `ubuntu-latest` is the same as `ubuntu-22.04`, at least until a new ubuntu VM rolls around (probably on 24.04).